edb/server/args.py

```diff
@@ -509,6 +554,17 @@ def _writer(status: str) -> None:
         envvar="EDGEDB_SERVER_INSTANCE_NAME",
         type=str, default=None, hidden=True,
         help='Server instance name.'),
+    click.option(
+        '--backend-capability-bound',
+        envvar="EDGEDB_SERVER_BACKEND_CAPABILITY_BOUND",
```

Comment:
I don't think this should be called a "BOUND", because, unlike systemd's `CapabilityBoundingSet`, we start from "all detected capabilities are enabled" and not from "all capabilities are disabled".  Let's simply call it "--backend-capabilities"